pkgs/games/anki/patches/skip-formatting-python-code.patch

```diff
@@ -1,31 +1,21 @@
-From 104572dc7ebb75061b867158ce3d4311d8cf4594 Mon Sep 17 00:00:00 2001
-From: Euan Kemp <euank@euank.com>
```

Comment:
Those patches were incompatible. I've recreated all patches manually in a local copy of the repository. What you see here is the output of `git format-patch -1 HEAD` (so, technically, the patch *is* created by me). The only thing unchanged is the idea of what to change, which I'm *not* trying to take credit for. After all, we track changes to the whole derivation, including patches.

I wonder if it makes sense to just remove the "From" field (two out of three patches didn't have it before) so as to remove the confusion.

Reply:
I personally prefer full patches (the workflow I used on my end of the update was: checkout anki 2.1.66, apply all 3 patches, git rebase to latest, resolve conflicts, `git format-patch HEAD^^^` to recreate the three commits -- so for the first step of applying the patches locally having full patch headers with a commit message/author allows using `git am` which is appreciable)
Using git rebase that way keeps the author that was in the patch, which doesn't change just because there were conflicts -- I think this makes most sense as the hard part is the idea of what to change, not the nitty gritty. (In theory if you want to claim your work on the patch you'd add `Co-authored-by` in the commit message body instead - you're correct in that we also have similar information in the nixpkgs git history but adding the patch doesn't mean you wrote it, many of the patches there have actually been written by someone else for the other project and just committed there, so I would only look there if the patch contains no author information)

Anyway, @euank himself didn't point it out so probably doesn't mind, and it's not that much extra work to apply the diffs manually if I ever need to work on this again so I have no strong feeling either way; let's keep plain diffs as you changed. Sorry for taking your time :)

Reply:
Thank you!